Make.inc

```diff
@@ -646,6 +646,10 @@ LLVM_CONFIG := $(build_bindir)/llvm-config$(EXE)
 endif
 endif # LLVM_CONFIG undefined
 
+ifeq ($(USE_SYSTEM_LLVM), 1)
+LLVM_VER = $(shell $(LLVM_CONFIG) --version)
+endif
```

Comment:
move the `JCPPFLAGS` from a few lines higher up into this same block? also there's a comment a few lines lower `endif # SYSTEM_LLVM` that isn't accurate any more